File: src/Belleza.java
public class Belleza extends Producto {
        private String marca;
        private double contenidoMl;
        public Belleza(String nombre, double precio, String fechaImportacion, int stock, String marca,double contenidoMl) {
            super(nombre, precio, fechaImportacion, stock);
            this.marca = marca;
            this.contenidoMl=contenidoMl;
        }

        @Override
        public String obtenerInformacion() {
            return super.obtenerInformacion() + ", marca: " + marca+",Contenido ml: "+contenidoMl;
        }

}

File: src/Belleza.py
from Producto import Producto

class Belleza(Producto):
    def __init__(self, nombre, precio, fecha_importacion, stock, marca, contenido_ml):
        super().__init__(nombre, precio, fecha_importacion, stock)
        self.marca = marca
        self.contenido_ml = contenido_ml

    def obtener_informacion(self):
        return f"{super().obtener_informacion()}, marca: {self.marca}, Contenido ml: {self.contenido_ml}"
